### DISCUSSION TITLE
How to express "search for `====`" in `config.toml` ?

### DISCUSSION THREAD

**Discussion by alibektas** [helix-editor/helix#9234] [Q&A] [answered]
As the title suggests I am looking for a way to do this. I would/could also appreciate if someone could tell me how I can go to the next merge conflict using Helix's built-in stuff ( not meaning mere search) but it would be also interesting to know if the thing I described can be expressed in the configuration file. 

An erroneous example 

```
[keys.normal."]"]
# r = ["search", "keyword"]
```

Thanks in advance

**Comment by TornaxO7** [comment]
> (...) how I can go to the next merge conflict (...)

I simply do `/<<<<<` or `/HEAD` for this

**Reply by alibektas** [reply]
Ofc that's one way of doing it but I was rather asking if there is a way to express this in the `config.toml`. 

EDIT : What I wanted was not clearly laid out in the question so I edited it, thank you.

**Comment by the-mikedavis** [comment, marked as answer]
This is not currently possible: it needs https://github.com/helix-editor/helix/issues/1383

**Reply by alibektas** [reply]
Thanks!
